### PR TITLE
Removed old Dark Theme, and replace it with the "Dark Grey"

### PR DESCRIPTION
**Pull request by ITAYC0HEN**
No more Dark theme with a blueish background :)

* Removed the old Dark Theme
* Turned the new "Dark Grey Theme" to the official "Dark Theme"
* More tiny modifications

### CODE REVIEW

File: src/utils/Configuration.cpp [hunk 1]
```diff
@@ -198,16 +193,16 @@ void Configuration::setFont(const QFont &font)
     emit fontsUpdated();
 }
 
-void Configuration::setDarkTheme(int theme)
+void Configuration::setTheme(int theme)
 {
-    s.setValue("dark", theme);
+    s.setValue("ColorPalette", theme);
```
Comment (xarkes) [inline src/utils/Configuration.cpp]: Why not use the `theme` termination?

File: src/utils/Configuration.cpp [hunk 2]
```diff
@@ -159,21 +159,9 @@ void Configuration::loadBaseDark()
...
-    setColor("gui.disass_selected", QColor(36, 66, 79));
-    // Disassembly line selected
-    setColor("highlight", QColor(64, 115, 115));
-    setColor("highlightWord", QColor(64, 115, 115));
-}
-
-void Configuration::loadDarkGreyTheme()
-{
-    loadBaseDark();
+    setColor("gui.background", QColor(37, 40, 43));    loadBaseDark();
```
Comment (xarkes) [inline src/utils/Configuration.cpp]: Can you put the loadBaseDark above?

File: src/utils/Configuration.cpp [hunk 3]
```diff
@@ -159,21 +159,9 @@ void Configuration::loadBaseDark()
 void Configuration::loadDarkTheme()
 {
     loadBaseDark();
-    setColor("gui.border",  QColor(255, 255, 255));
+    setColor("gui.border",  QColor(100,100,100));
```
Comment (xarkes) [inline src/utils/Configuration.cpp]: Can you add spaces after commas?

File: src/utils/Configuration.cpp (hunk 1 above)
Comment (ITAYC0HEN) [reply src/utils/Configuration.cpp]: Already used for radare2 text themes ("eco"...)

**Pull request merged**